### PR TITLE
Update port.rb Documentation

### PR DESCRIPTION
**Pull request by nvtkaszpir**
fix comment section, expand example section

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:
- @nvtkaszpir

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: lib/resources/port.rb [hunk 1]
```diff
@@ -8,7 +8,8 @@
 # Usage:
```
Comment (chris-rock) [inline lib/resources/port.rb]: Since we are at it, lets remove the usage comment, since everything is covered within the `example` block

**Comment by chris-rock** [comment]
@nvtkaszpir Thanks for fixing that. Please do not forget to sign the CLA

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

**Comment by chris-rock** [comment]
Thanks a lot for the improvement @nvtkaszpir

**Pull request merged**